<!DOCTYPE html>
<html lang="en" dir="ltr">

<head>
    <meta charset="utf-8">
    <title>Predicted interactions to iteratively update species distribution models</title>
    <script src="https://hypothes.is/embed.js" async></script>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
    integrity="sha256-l85OmPOjvil/SOvVt3HnSSjzF1TUMyT9eV0c2BzEGzU=" crossorigin="anonymous" />
    <style>
        @import url('https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,200;0,300;0,400;0,600;0,700;0,800;0,900;1,200;1,300;1,400;1,600;1,700;1,800;1,900&display=swap');
                @import url('https://fonts.googleapis.com/css2?family=Cousine:ital,wght@0,400;0,700;1,400;1,700&display=swap');
                :root {
                    --structure: #1976D2;
                    --links: #00897B;
                    --light: #9E9E9E;
                    --pale: #e0e0e0;
                }
                body {
                    font-family: "Nunito", sans-serif;
                    margin: 0 0;
                    padding: 0 0;
                }
                a {
                    text-decoration: none;
                    color: var(--links);
                }
                a:hover {
                    font-weight: 600;
                }
                .container {
                    margin: 0em auto;
                    max-width: 1000px;
                    padding: 1em;
                }
                code, pre {
                    font-family: "Cousine", monospace;
                }
                pre {
                    white-space: pre-wrap;
                }
                h1, h2, h3 {
                    color: var(--structure);
                }
                header {
                    position: fixed;
                    top: 0;
                    left: 0;
                    width: 100vw;
                    background-color: var(--structure);
                    z-index: 5;
                    color: white;
                }
                .header--title {
                    font-weight: 900;
                    text-transform: uppercase;
                }
                .header--authors > span > a {
                    color: white;
                    font-size: 90%;
                }
                main {
                    padding-top: 8em !important;
                    padding-bottom: 8em !important;
                }
                footer {
                    background-color: var(--pale);
                }
                .abstract {
                    color: var(--light);
                    margin-bottom: 5em;
                }    </style>
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.10.0-rc.1/dist/katex.min.css"
    integrity="sha384-D+9gmBxUQogRLqvARvNLmA9hS2x//eK1FhVb9PiU86gmcrBrJAQT8okdJ4LMp2uv" crossorigin="anonymous" />
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.10.0-rc.1/dist/katex.min.js"
    integrity="sha384-483A6DwYfKeDa0Q52fJmxFXkcPCFfnXMoXblOkJ4JcA8zATN6Tm78UNL72AKk+0O"
    crossorigin="anonymous"></script>
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.10.0-rc.1/dist/contrib/auto-render.min.js"
    integrity="sha384-yACMu8JWxKzSp/C1YV86pzGiQ/l1YUfE8oPuahJQxzehAjEt2GiQuy/BIvl9KyeF"
    crossorigin="anonymous"></script>
    <script>
        document.addEventListener("DOMContentLoaded", function () {
            renderMathInElement(document.body, {
                delimiters: [
                { left: "$$", right: "$$", display: true },
                { left: "$", right: "$", display: false }
                ]
            });
        });
    </script>
</head>

<body>
    <header>
        <div class="container">
            <div class="header--title">Predicted interactions to iteratively update species distribution models</div>
            <div class="header--authors">
                                <span class="header--author">
                                        <a href="https://orcid.org/0000-0003-2791-8383" target="_blank">
                        Gracielle Higino
                    </a>
                                                                <sup class="affilcode">1</sup>
                                    </span>
                                <span class="header--author">
                                        <a href="https://orcid.org/0000-0001-9051-0597" target="_blank">
                        Francis Banville
                    </a>
                                                                <sup class="affilcode">2</sup>
                                            <sup class="affilcode">3</sup>
                                            <sup class="affilcode">4</sup>
                                    </span>
                                <span class="header--author">
                                        <a href="https://orcid.org/0000-0002-2212-3584" target="_blank">
                        Gabriel Dansereau
                    </a>
                                                                <sup class="affilcode">2</sup>
                                            <sup class="affilcode">4</sup>
                                    </span>
                                <span class="header--author">
                                        <a href="https://orcid.org/0000-0002-0735-5184" target="_blank">
                        Timothée Poisot
                    </a>
                                                                <sup class="affilcode">2</sup>
                                            <sup class="affilcode">4</sup>
                                    </span>
                            </div>
        </div>
    </header>

    <main class="container">
        <section class="abstract">
            <b>Abstract:</b> Species distribution models are characterized by three aspects of a species occurrence: its biotic environment, its abiotic environment, and its mobility range. Nevertheless, most distribution models do not address the biotic interactions that potentially shape a species range, thus measuring only how adequate an environment is for a given population. In this chapter we discuss how we can work towards better species distribution models without ignoring our knowledge about ecological networks and communities’ assemblage. We suggest this can be done in seven steps: 1. careful selection of species to be modelled; 2. performance of habitat suitability models; 3. prediction of a metaweb which contains the selected species; 4. prediction of local networks’ structure and assessment of their feasibility; 5. adjustment of the probabilities of occurrence based on the probabilities of realization of networks; and 6. iteration of this process until the differences between probabilities tends to zero. We highlight how this can be done with the help of machine learning techniques both to predict local networks and to update the results of grinellian habitat suitability models. Furthermore, we point to promising directions on the development of these techniques and the main challenges ecologists might face in the near future.<br />
            <div class="artifacts">
                <a href="draft.pdf" title="Draft PDF" target="_blank">draft</a>
                <a href="preprint.pdf" title="Preprint PDF" target="_blank">preprint</a>
            </div>
        </section>

        <article>
            <h2 id="introduction">Introduction</h2>
            <p>The occurrence of a species in a given location is an encrypted message that travels through time. It carries the species’ evolutionary history, long migration journeys, effects of other species we do not even know that exist, and ultimately the elements that shape its, yet unknown, future. Ecologists have been trying to decode this message with progressively more powerful tools, since their own field notes to highly complex computational algorithms, such as habitat suitability models. These models were born as an attempt to model species’ distribution based on their niche, considering their occurrences as sample points of suitable abiotic variables and their absences as sample points of unsuitable variables. However, these observations (environmental variables and geographic location) only unveils part of the mystery, and the missing link are ecological interactions.</p>
            <p>Species distribution models can be untangled in three aspects of a species occurrence: its biotic environment - the connections it makes with other species -, its abiotic environment - the connection it makes with non-living resources -, and its mobility range - how far it can go (fig. 1)<span class="citation" data-cites="Peterson2012EcoNic">(Peterson et al. 2012)</span>. The biotic environment act on these models as potential and realized interactions, constrained or enabled by abiotic factors, geographical conformation and migratory ability. Nevertheless, most distribution models do not address the biotic interactions that potentially shape a species range (or rather do so indirectly), thus measuring only how adequate an environment is for a given population. Because of that, they are denominated Habitat Suitability Models (hereafter HSMs).</p>
            <figure>
            <img src="figures/bam.png" id="fig:bam" alt="Figure 1: The “BAM diagram,” adapted from (Jorge Soberón 2007). Open circles are absences and solid circles are observed presences. Big circles correspond to the theoretical space of a species, regarding its biotic interactions (the B), the abiotically suitable space (the A) and the geographic area accessible to it (the M). These three aspects represent real points of occurrence on the real geographic space (the G). Ecological interactions act over this model in four ways: in (1), there are potential interactions that are never realized because of geographical and environmental constraints; in (2) interactions are realized on accessible, abiotically suitable areas; the space (3) is where the species could eventually go and establish new interactions, while (4) is the area where the occurrence of the species is limited only by abiotic factors." /><figcaption aria-hidden="true">Figure 1: The “BAM diagram,” adapted from <span class="citation" data-cites="JorgeSoberon2007GriElt">(Jorge Soberón 2007)</span>. Open circles are absences and solid circles are observed presences. Big circles correspond to the theoretical space of a species, regarding its biotic interactions (the B), the abiotically suitable space (the A) and the geographic area accessible to it (the M). These three aspects represent real points of occurrence on the real geographic space (the G). Ecological interactions act over this model in four ways: in (1), there are potential interactions that are never realized because of geographical and environmental constraints; in (2) interactions are realized on accessible, abiotically suitable areas; the space (3) is where the species could eventually go and establish new interactions, while (4) is the area where the occurrence of the species is limited only by abiotic factors.</figcaption>
            </figure>
            <p>The use of HSMs is very convenient because environmental variables and geographic limits are not (highly) dynamic variables from the evolutionary point of view. Because the climate (used to) change at a very slow pace, as well as species’ niche, we could expect to find the same pool of species that are able to live in a certain region, even if populations fluctuated at a smaller temporal scale. This is because the cumulative effect of small scale variation on climate, population dynamics and habitat suitability itself results in macroecological outcomes such as combinations of extinction and cladogenesis, which lead to biodiversity distribution at continental scales. Also, abiotic variables are not under the influence of the focal species, which make them statistically safe, and their relationship with the species’ niche is assumed to be static in space and time, which adds generalization to the model. The biotic space, on the other hand, is usually highly dynamic and variable, and it can be stochastic at very small scales to predictable structures at large scales. Additionally, because ecological networks are the cumulative result of local events <span class="citation" data-cites="Poisot2016HowEco Guimaraes2020StrEco">(Poisot and Stouffer 2016; Guimarães 2020)</span>, its properties can vary with environmental factors and species evolutionary history <span class="citation" data-cites="MartinGonzalez2015MacPhy Dalsgaard2013HisCli">(Martín González et al. 2015; Dalsgaard et al. 2013)</span>.</p>
            <p>There is a big ecological and evolutionary leap between local dynamics of species and the biogeographical processes that are the primary assumptions to the habitat suitability and species distribution models. However, because ecological networks are very informative and aggregate populations’ dynamics through scales, it is conceptually important to include them in HSMs. In fact, it has been shown that HSMs are more efficient when ecological interactions are accounted for (either directly or indirectly) <span class="citation" data-cites="Wisz2013RolBio Cazelles2016IntBio">(Wisz et al. 2013; Cazelles et al. 2016)</span>. Some strategies have been adopted by the scientific community to accomplish that and are shortly reviewed later in this paper. Correlative approaches assume that the co-ocurrence of related species accounts for interaction, while mechanistic models try to refine this assumption by species traits and phenology. Currently, the scenario of habitat suitability models accounting for the biotic environment is either too generalistic (correlative approaches) or too precise (mechanistic), in the sense that they only work when we have a good amount of information about that specific species. However, empirical data on ecological interactions are scarce, and, on the other hand, we cannot just assume that two species will always interact when they co-occur. How could we find balance and go further?</p>
            <p>The good news is that ecologists have been developing techniques to predict and forecast the ecologically realistic number of links <span class="citation" data-cites="MacDonald2020RevLin">(MacDonald, Banville, and Poisot 2020)</span>, the nature of ecological interactions <span class="citation" data-cites="Elmasri2020HieBay">(Elmasri et al. 2020)</span>, and networks’ properties with good accuracy. These techniques can mitigate the large and biased eltonian shortfall that we have now <span class="citation" data-cites="Poisot2020EnvBia Hortal2015SevSho">(Poisot et al. 2020; Hortal et al. 2015)</span>. In this context, we can envision an integrative approach of species distribution modelling combined with network prediction resulting in a more realistic, yet generalist, model where the predicted networks update the probabilities of occurrence computed by an HSM. In this paper we invite you to envision better species distribution models, which do not ignore our knowledge about ecological networks and communities assemblage. Here we suggest this can be done with the help of machine learning techniques both to predict local networks and to update the results of grinellian HSMs. We point to promising directions on the development of these techniques and main challenges ecologists might face in the near future.</p>
            <figure>
            <img src="figures/concept.png" alt="TODO" /><figcaption aria-hidden="true">TODO</figcaption>
            </figure>
            <h2 id="hsms-the-mechanics-innovations-and-drawbacks">HSMs: the mechanics, innovations and drawbacks</h2>
            <p>Habitat suitability models aim at finding relationships between the occurrence of species and their environment <span class="citation" data-cites="Guisan2000PreHab Guisan2017HabSui">(Guisan and Zimmermann 2000; Guisan, Thuiller, and Zimmermann 2017)</span>. The reader might have encountered different terminologies in this area, such as species distribution modelling and ecological niche modelling, that supposedly have the same objectives. The terminology is a matter of debate on the scientific community, and here we chose to distinguish habitat suitability models from species distribution models.</p>
            <p>Ecological niche models and habitat suitability models focus on the the area of A of the BAM diagram fig. 1 where a species can occur, which means they calculate the fundamental niche of species <span class="citation" data-cites="Peterson2012EcoNic">(Peterson et al. 2012)</span>. This can be achieved by finding the relationship between environmental conditions and the presence or absence of a certain species. This relationship can be static or dynamic in space, and only makes sense when calculated for the area inside M. Therefore, this means that they will find suitable areas inside the area 2, where species really are, but can also find suitable areas in 5, where the species probably are not because of biotic unsuitability. Species distribution models, on the other hand, should aim at modelling 2 (B <span class="math inline">∩</span> A <span class="math inline">∩</span> M), which means considering biotic constraints <span class="citation" data-cites="Peterson2012EcoNic">(Peterson et al. 2012)</span>. Although they rarely do so, Guisan and Zimmermann (2000) argue that the observation data used as input on these models carries these information: when we use the physiological limits of a species as a variable to be correlated to the environment, we are modelling its fundamental niche, while using observational field occurrence data implies that we are modelling the realized niche (thus, the species distribution) because these data implicitly accounts for biotic limitations <span class="citation" data-cites="Guisan2000PreHab">(Guisan and Zimmermann 2000)</span>. However, because the biotic constraints are not explicitly considered in the models, it is possible that the predicted distribution area reaches places with completely different communities. Because we did not consider previous knowledge about species interactions, how can we interpret this result? Given these differences between SDMs and HSMs, many statistical approaches can be used to model both of them. Here we focus on the most innovative algorithms to find the species’ suitable habitats, to further develop ideas on how to integrate biotic constraints.</p>
            <p>Habitat suitability models are built over five steps: conceptualization, data sampling, calibration, evaluation and prediction <span class="citation" data-cites="Guisan2017HabSui">(Guisan, Thuiller, and Zimmermann 2017)</span>. The conceptualization is related to the core ideas of niche, as illustrated in fig. 1, but also to the main goals of the investigation. What processes are important to the patterns we a dealing with, what is the frequency in which they repeat in space (determining the grain and extent of the environmental variables) and what are the direct and indirect predictors are examples of questions that should be asked at this step. The answers to these questions will determine how the data should be sampled (the spatial scale of the variables) and the appropriate selection of the variables <span class="citation" data-cites="Guisan2000PreHab">(Guisan and Zimmermann 2000)</span>. Then the actual building of the mathematical model enters the stage, with iterative adjustments to capture its variations. The data is separated in training validation sets, and then the model is used to predict and project probabilities is space that can be interpreted as habitat suitability.</p>
            <!---innovations--->
            <p>While statistical inference is a telescope that allows us to investigate the past of biodiversity structure, machine learning algorithms are some of the time machines we use to assess the past in order to predict the future. They have the power to learn from the data, identifying structure and generating predictions <span class="citation" data-cites="Olden2008MacLea">(Olden, Lawler, and Poff 2008)</span>. One sound advantage of these models compared to statistical inference is that they allow us to explore “mischievous” data, embracing the multiple frequency distributions we find in nature. Many of them have become popular among ecologists, such as MaxEnt <span class="citation" data-cites="Phillips2006MaxEnt">(Phillips, Anderson, and Schapire 2006)</span> and Random Forest <span class="citation" data-cites="Breiman2001RanFor">(Breiman 2001)</span>, while making a good match with the continuously growing amount of biodiversity data. Although these models call for cautious usage for their complexity (and hard to interpret outputs) <span class="citation" data-cites="Rangel2012LabEco">(Rangel and Loyola 2012)</span>, it also demands some ecological knowledge about the relationships being modelled and careful parameterisation of terms. These very characteristics are the ones that allow us to explore nuances of biodiversity relationships, such as the role of ecological interactions in shaping species distribution - i.e. putting the B back in the BAM model.</p>
            <!---drawbacks--->
            <p>However, these time machines need a past to learn from, and the lack of information about species interactions makes this past quite blurry. Ecological interactions occur across taxonomic and geographic scales <span class="citation" data-cites="Guimaraes2020StrEco">(Guimarães 2020)</span>, and the task to collect these information through large extents and in fine grain (both temporal and spatial) is daunting. In addition, it is known that networks vary with climate, in space and with phylogenetic distance [<span class="citation" data-cites="Poisot2014SpeWhy">Poisot, Stouffer, and Gravel (2014)</span>; CITATIONS]. Despite the dynamic nature of networks’ structures (that are predictable outcomes of common processes), methods such as stacked (S-SDMs) and joint Species Distribution Models (JSDMs) are very common to address the influence of one species over another [CITATIONS]. These methods assume that, if a pair of species are known to interact, they will always do so once they co-occur [CITATIONS]. For example, in order to do a stacked SDM, one can model the assembly as a single unit, use the predicted distribution of each species and pile them up together, or model species at the same time, but as separate units <span class="citation" data-cites="Ferrier2006SpaMod">(Ferrier and Guisan 2006)</span>. On the other hand, JSDMs use a hierarchical regression model to calculate the probability of occurrence of all species together in response to environmental variables <span class="citation" data-cites="Pollock2014UndCoo">(Pollock et al. 2014)</span>. Both approaches can be considered great advances regarding previous methods that simply added the distribution of one or more other species as limiting factors in the model [CITATIONS]. S-SDMs and JSDMs try to incorporate another level of complexity by modelling assembly rules as an intermediate step towards more realistic HSMs (see <span class="citation" data-cites="Zurell2020TesSpe">Zurell et al. (2020)</span> for a comparison between both methods). Nevertheless, our time machines can take us even further and incorporate actual species interaction and predicted network properties in species distribution models.</p>
            <h2 id="going-further">Going further</h2>
            <!---why interactions in SDMs--->
            <p>Spatially structured co-occurrence can be the result of many historical and geographical processes, such as environmental heterogeneity and the similarity of environmental preferences between species <span class="citation" data-cites="Bar-Massada2017NonCoo">(Bar-Massada and Belmaker 2017)</span>. HSMs that account for co-occurrence as a proxy for interactions might wrongly capture these processes as biotic interactions <span class="citation" data-cites="Blanchet2020CooNot">(Blanchet, Cazelles, and Gravel 2020)</span>. An important analysis to be made before going further is at which scale ecological interactions influence the distribution of species. Although empirical evidence exists that ecological interactions are important processes on population arrangements in space [CITATIONS] and networks variation can be detected in macroecological scales [CITATIONS], their precise relationship with spatial scale remains unclear. A second challenge is to deal with highly dynamic properties of ecological interactions. How can we account for dependency between local processes in macroecological scales? We argue that Artificial Intelligence (AI) tools can help us identify structure on the available data and predict interactions and networks for communities we did not access yet.</p>
            <h3 id="the-eltonian-noise-paradox">The Eltonian Noise paradox</h3>
            <p>The Eltonian Noise Hypothesis states that the effect of ecological interactions on species distribution is captured at coarse resolution analyses as a covariation with environmental factors <span class="citation" data-cites="Soberon2009NicDis">(Soberón and Nakamura 2009)</span>. This logic reverts as an assumption that always that two given species co-occur, they will interact, which has been demonstrated to be not true <span class="citation" data-cites="Blanchet2020CooNot">(Blanchet, Cazelles, and Gravel 2020)</span>. In fact, the contrary is likely to be true: the nature of interactions happening at a given location affects co-occurrence <span class="citation" data-cites="Bullock2000GeoSep Chesson2008IntPre Godsoe2012HowSpe Svenning2014InfInt Godsoe2017IntBio">(Bullock et al. 2000; Chesson and Kuang 2008; Godsoe and Harmon 2012; Svenning et al. 2014; Godsoe et al. 2017)</span>, and they vary in space and time due to climate change, phylogenetic diversity (addition or extinction of clades in the community, for example)<span class="citation" data-cites="Davies2011PhyDiv">(Davies and Buckley 2011)</span>, population density <span class="citation" data-cites="Carnicer2009TemDyn">(Carnicer, Jordano, and Melián 2009)</span>, and many other factors. Additionally, analyses of range expansion rates - a common and very important application of HSMs - are intrinsically connected to interspecific interactions at the border of the ranges: the expansion tends to be slower when generalists predators are present or when mutualists are absent <span class="citation" data-cites="Svenning2014InfInt">(Svenning et al. 2014)</span>. On the other hand, range preservation is also associated with ecological interactions, once connected species can be protected of climate change and invasion <span class="citation" data-cites="Dunne2002NetStr Memmott2004TolPol Ramos-Jiliberto2012TopPla">(Dunne, Williams, and Martinez 2002; Memmott, Waser, and Price 2004; Ramos-Jiliberto et al. 2012)</span>.</p>
            <p>This hypothesis also seems to find no support when we investigate the betadiversity of links in ecological networks. In parasite-hosts systems in Eurasia, it is possible to detect a distinct difference in interactions between shared species in macroecological scale, suggesting an “unpacking” of species when biodiversity is expected to be lower [CHAPTER 1]. In addition, it is likely that species with larger ranges of distribution and those that are more generalists would co-occur with a greater number of other species <span class="citation" data-cites="Dattilo2020SpeDri">(Dáttilo et al. 2020)</span>, while dispersal capacity of competitive species modulate their aggregation in space and the effect of interactions on their range limits <span class="citation" data-cites="Godsoe2017IntBio">(Godsoe et al. 2017)</span>.</p>
            <p>Certainly, the exact macro scale effect of interspecies interactions might depend on the nature of these interactions, the system being studied and the combination of geographical and environmental factors. Younger communities may be more affected by environmental limitations, once they are dominated by generalist species, while older metacommunities are probably affected in different ways in the centre of the distribution, at the edge of ranges and in sink and source communities. In fact, there is a growing number of evidence demonstrating that biotic interactions are important factors modulating range shift and expansion due to rapid climate change (where evolutionary processes are ignored, assuming that species will not have time to adapt) <span class="citation" data-cites="Bullock2000GeoSep Hellmann2012InfSpe Afkhami2014MutEff Godsoe2017IntBio Siren2020IntRan">(Bullock et al. 2000; Hellmann, Prior, and Pelini 2012; Afkhami, McIntyre, and Strauss 2014; Godsoe et al. 2017; Sirén and Morelli 2020)</span>.</p>
            <p>Modelling species distribution considering the Eltonian Noise Hypothesis to be true can be very useful to understand general structures of biodiversity. At coarse resolutions, with dubious quality and sparse data, and when the space for error is large, these models are probably the best we can do. However, no species is an island and we should go further whenever possible, investigating whether and how ecological interactions exert important pressures on species ranges and spatial arrangements. This will help us avoid spurious inferences, especially when these models aim at conservation strategies. Fortunately, the data we need to do that has been increasingly available <span class="citation" data-cites="Konig2019BioDat">(König et al. 2019)</span>, and technologies to deal with them (with their biases and gaps) are following.</p>
            <h3 id="non-stationarity-of-interactions-and-networks-in-space-time-and-across-resolutions">Non-stationarity of interactions and networks in space, time and across resolutions</h3>
            <p>Ecological interactions are dynamic entities of ecosystems that vary in space and time. The reason for this fluctuability is a combination of factors that shape the probability of interactions: environmental changes that affect the metabolism of individuals and the abundance of populations <span class="citation" data-cites="Rall2012UniTem Poisot2016StrPro Muola2010AssPla">(Rall et al. 2012; Poisot, Cirtwill, et al. 2016; Muola et al. 2010)</span>, changes in habitat <span class="citation" data-cites="Tylianakis2017EcoNet">(Tylianakis and Morris 2017)</span>, and the phylogenetic structure of communities <span class="citation" data-cites="Coelho2017NeuBio">(Coelho, Rodrigues, and Rangel 2017)</span>. Notably, the variation in traits distribution (including phenological mismatches) are important drivers of variation in species interactions [CITATIONS].</p>
            <p>Ecological networks reflect these dynamics, and, as a result, varies in space not always in the same fashion that species richness do <span class="citation" data-cites="Poisot2014SpeWhy">(Poisot, Stouffer, and Gravel 2014)</span>. The non-stationarity of ecological networks and their relationship with the environment can be measured and have been demonstrated in the last few years. <span class="citation" data-cites="Dalsgaard2013HisCli">Dalsgaard et al. (2013)</span> found that modularity can be reduced and nestedness increased due to climate change speed <span class="citation" data-cites="Dalsgaard2013HisCli">(Dalsgaard et al. 2013)</span>. The phylogenetic diversity of networks is affected by dispersion and speciation rates <span class="citation" data-cites="Coelho2017NeuBio Sebastian-Gonzalez2015MacTre Trojelsgaard2013MacPol">(Coelho, Rodrigues, and Rangel 2017; Sebastián-González et al. 2015; Trøjelsgaard and Olesen 2013)</span>, characteristics highly correlated to both traits and geographical structure of the environment. These examples illustrate how networks can vary both in space (due to environmental filtering of species traits, for example) and time (due to evolutionary changes and phenology).</p>
            <p>Interestingly, because ecological networks are the cumulative pool of local interactions, it is possible that network structure varies across scales <span class="citation" data-cites="Galiana2018SpaSca">(Galiana et al. 2018)</span>. For example, increased species richness can promote species packing and specialization of interactions at local scale even for generalist species [CITATIONS; CHAP 1]. Moreover, the local combinations of species and interactions in a community can result in very different regional pictures, since interactions can affect one another. For example <span class="citation" data-cites="Sanders2012IndCom">Sanders and Van Veen (2012)</span> observed that the persistence of two species of wasps separated by four trophic links are conditioned to one another because they regulate the population of competitive aphids <span class="citation" data-cites="Sanders2012IndCom">(Sanders and Van Veen 2012)</span>. The replacement of one species in this network can result in the local extinction of another, even if they are not directly linked, and this dynamic could be easily misinterpreted in regional scales as an environmental constraint.</p>
            <p>All these peculiarities about ecological interactions and networks can be faced as challenges or opportunities for the next years. Once we understand how and where interactions are important to the accuracy of species distribution models, we can use the dynamics described here as refinements on our models. On the other hand, some of these mechanisms act at a scale so small that stochasticity takes the stage and can be accounted for in the error term of most models. This trade-off reinforces the need to carefully adjust distribution models according to the system being studied.</p>
            <h3 id="why-we-can-go-further">Why we can go further</h3>
            <p>A combination of global efforts and technology can make us take another step in combining species occurrence data and ecological interactions. In the last decades we have witnessed an increase in biodiversity data, including citizen-science projects <span class="citation" data-cites="Pocock2015BioRec Callaghan2019ImpBig">(Pocock et al. 2015; Callaghan et al. 2019)</span> and organized dedicated databases mostly accessed by specialists, such as mangal and GBIF <span class="citation" data-cites="Poisot2016ManMak Gbi">(Poisot, Baiser, et al. 2016; <span>“GBIF,”</span> n.d.)</span>. These data frequently describe the occurrence and the taxonomic identity of a species, but they also capture real-time interactions <span class="citation" data-cites="Roy2016FocPla Ryan2018RolCit">(Roy et al. 2016; Ryan et al. 2018)</span>. Moreover, both occurrence and interaction data share common characteristics that can be learned and subsequently inferred. Because species occurrence are subject to their evolutionary history that shaped their ecological niche, one can roughly infer where it could occur only based on known occurrence point locations (that is how HSMs work, as described earlier in this paper). This is connected to how ecological networks also show signs of common ancestry and preserve a basic structure <span class="citation" data-cites="Riva2016ExpEvo Dallas2018ComTur">(Riva and Stouffer 2016; Dallas and Poisot 2018)</span>. The ever-growing number of available data and detectable structure on ecological networks are the main ingredients we need to develop mathematical models capable of learning from what we already know to predict what we do not know yet.</p>
            <h4 id="filling-in-eltonian-gaps-with-mathematics-predicting-interactions-networks-and-doing-it-across-scales">Filling in eltonian gaps with mathematics: predicting interactions, networks, and doing it across scales</h4>
            <p>A complete assessment of ecological interactions is even more difficult than sufficient sampling of biodiversity. The number of interactions sampled will always be lower than the number of possible interactions, mainly due to the existence of forbidden links <span class="citation" data-cites="Jordano2016SamNet">(Jordano 2016)</span>. This lack of information, known as the Eltonian Shortfall, is aggravated by biases and differences in sampling methods <span class="citation" data-cites="Hortal2015SevSho">(Hortal et al. 2015)</span>. Nevertheless, in the same way as the knowledge about the natural history of organisms help us validate HSMs, understanding a species’ behaviour, traits and phylogenetic relationships can help us identify if the lack of links sampled is due to insufficient effort or natural mismatch between species. Additionally, interactions vary in space and time, which adds to the uncertainty of an unregistered interaction. Hence, rather than assuming interactions to be binary variables, we can think of them as probabilities <span class="citation" data-cites="Poisot2016StrPro">(Poisot, Cirtwill, et al. 2016)</span>. In fact, assuming interactions as probabilistic events is more of an opportunity than an obstacle. Once we understand what is the basic mix than can result in a connection between ecological units (from individuals to ecosystems), we can use probabilities to estimate the likelihood of our ideas even when data is lacking. Usually, this basic mix is composed by abundance, co-occurrence in space and time (conditional to a given environment), and traits matching [<span class="citation" data-cites="Poisot2014SpeWhy">Poisot, Stouffer, and Gravel (2014)</span>; CITATIONS]. The regional pool of interactions also plays an important role on interactions realization through indirect effect <span class="citation" data-cites="Sanders2012IndCom Poisot2014SpeWhy">(Sanders and Van Veen 2012; Poisot, Stouffer, and Gravel 2014)</span>, but they also add another layer of complexity in predictive models.</p>
            <p>When we do not have information about any of these properties, interactions tend to behave as stochastic elements in our model, while the more information we have, the more our equations will look like a niche model. Maybe one of the most neutral approaches to predicting interactions is assuming that two species will always interact once they co-occur. On possible step forward is to account for spatial structure of species distribution: interactions are more or less likely to occur according to the location of the population inside the species range <span class="citation" data-cites="Svenning2014InfInt Godsoe2017IntBio Bar-Massada2017NonCoo">(Svenning et al. 2014; Godsoe et al. 2017; Bar-Massada and Belmaker 2017)</span>. Moving away from stochasticity a little bit, <span class="citation" data-cites="Canard2012EmeStr">Canard et al. (2012)</span> illustrates how we can build neutral networks based on species abundance and richness, and found that the emergent properties of these networks are compatible with empirical ones. With this method, it is possible to identify neutral forbidden links <span class="citation" data-cites="Canard2012EmeStr">(Canard et al. 2012)</span>, which unfolds many other questions that can be answered without previous empirical data. For example, we can investigate whether abundance fluctuation is the main driver of betadiversity of links between networks, or compare how networks are different from the neutral model when only the traits vary. It is important to notice, however, that abundance also plays a role on the nature of the interaction. Mutualistic interactions, for example, can turn into parasitism or competition when one of the players is very abundant <span class="citation" data-cites="Wolin1984ModFac">(Wolin and Lawlor 1984)</span>.</p>
            <p>An intermediate step could be to find detectable interactions in a species pool. <span class="citation" data-cites="XiaoFu2019LinPre">Xiao Fu et al. (2019)</span> proposed a combination of a Poisson N-mixture model and collaborative filtering to predict potential links under imperfect detection. This machine learning model allowed them to successfully infer interactions based on a few observed occurrences sampled in the field <span class="citation" data-cites="XiaoFu2019LinPre">(Xiao Fu et al. 2019)</span>. This approach deals with interactions as countable units, ignoring evolutive and ecological mechanisms that could lead do variation in the probability of their realization. However, it can be a good exploratory method to investigate the completeness of interaction sampling.</p>
            <p>On our way to more mechanistic models, we can start looking for evolutionary clues on our encrypted message. First, we can assume that similar species will interact in similar ways with their “pairs” due to phylogenetic inertia <span class="citation" data-cites="Gomez2010EcoInt Peralta2016MerEvo">(Gómez, Verdú, and Perfectti 2010; Peralta 2016)</span>. If predators <em>A</em> and <em>B</em> are closely related, they are likely to compete for the same set of preys. In this sense, we can use machine learning algorithms to find in the “potential interactions” pool those that are more ecologically likely to occur. For example, <span class="citation" data-cites="Desjardins-Proulx2017EcoInt">Desjardins-Proulx et al. (2017)</span> showed that the <em>K</em> nearest neighbour (KNN) and the Random Forest (RF) algorithms are complementary when it comes to identifying the probability of a predator to interact with a certain prey and binary occurrence of interactions (RF predicts either interaction or non-interaction) <span class="citation" data-cites="Desjardins-Proulx2017EcoInt">(Desjardins-Proulx et al. 2017)</span>. According to the authors, the usefulness of these methods depend on the kind of data we have in hands: KNNs need previous information about interactions to learn from, while RFs should use a set of traits. Another example of link prediction based on trait matching is <span class="citation" data-cites="Dallas2017PreCry">Dallas, Park, and Drake (2017)</span>, where the authors identified cryptic associations between hosts and parasites based on a Bayesian model. Similar to the approach on <span class="citation" data-cites="XiaoFu2019LinPre">Xiao Fu et al. (2019)</span>, this model also allow us to investigate the completeness of interaction sampling, but now based on traits matching and not only on occurrence observations. <span class="citation" data-cites="Pichler2019MacLea">Pichler et al.</span> provide an extensive methodological comparison of Machine Learning models to predict interactions based on traits matching. Nonetheless, these techniques isolate interactions from their biotic and abiotic environments, and this should be kept in mind whenever predictions are made.</p>
            <p>Another step further would be implementing evolutionary models in the relationship between links and phylogeny. <span class="citation" data-cites="Elmasri2020HieBay">Elmasri et al. (2020)</span> point out that recent approaches to link prediction using phylogenetic similarity assume a linear or fixed diversification rate. They proceed to develop a phylogenetic matching model of predicted interactions using the early-burst tree scaling model, where the traits diversification tree is transformed, adjusted by a single parameter, either to early or late diversification <span class="citation" data-cites="Harmon2010EarBur Elmasri2020HieBay">(Harmon et al. 2010; Elmasri et al. 2020)</span>. The authors also accounted for uncertainty in unobserved interactions, which improved the accuracy of the predictions. The adjustment of the early-burst parameter can be based on ecological reasoning. For example, food webs with species clustered in trophic levels (specialized predators, for example) seems to have a trait evolution with a high diversification rate early in the tree, while networks with many omnivorous species are thought to have a late diversification of traits <span class="citation" data-cites="Ingram2012WheSho">(Ingram, Harmon, and Shurin 2012)</span>. These predictions are highly mechanistic, accounting for evolution and ecology of interactions.</p>
            <p>Connected to the Eltonian Shortfall described at the beginning of this session, there are global gaps and biases on network sampling <span class="citation" data-cites="Poisot2020EnvBia">(Poisot et al. 2020)</span>. Because ecological networks are made of nodes and edges, and both can be treated as probabilities (addressing the occurrence of a node and the realization of an interaction), we can investigate the chances of different networks to emerge from a set of species co-occurring in a given location <span class="citation" data-cites="Poisot2016StrPro">(Poisot, Cirtwill, et al. 2016)</span>. Recently, <span class="citation" data-cites="MacDonald2020RevLin">MacDonald, Banville, and Poisot (2020)</span> demonstrated how we can estimate the number of possible links in a network based on the number of species it has, which take us closer to predict networks themselves since we have much more information about species than we have on interactions <span class="citation" data-cites="MacDonald2020RevLin">(MacDonald, Banville, and Poisot 2020)</span>. Knowing the number of possible links given the species richness in a given location allow us to calculate the probability distribution for each network property, which can be further connected to what we know about the relationship between networks’ properties and species spatial distribution</p>
            <p>One question that is frequently stressed by macroecologists is how to connect local processes, such as interactions, to large scale biodiversity structures. The sampling of both interactions and environmental variables are usually made in opposing ends of spatial resolution, which makes them incomparable from a statistical point of view <span class="citation" data-cites="Peterson2012EcoNic">(Peterson et al. 2012)</span>. Also, because of the nature of the sampling, it is hard to transform those variables to make them vary in compatible scales. How can we deal with this problem when predicting interactions and networks for macroscales? In fact, the basic mix components that leads to the occurrence of an interaction varies with the spatial resolution in which it is assessed. Consequently, the distribution of probabilities for interactions and network properties would change, possibly leading to different interpretation of biodiversity distribution. However, the different ways in which networks properties vary through scales have also been investigated, and simulations can help us understand that more deeply <span class="citation" data-cites="Poisot2014SpeWhy Peralta2016MerEvo Guimaraes2020StrEco">(Poisot, Stouffer, and Gravel 2014; Peralta 2016; Guimarães 2020)</span>. <em>Because</em> interactions vary more in space than species occurrence (while adding to this information) <span class="citation" data-cites="Poisot2017HosPar">(Poisot et al. 2017)</span>, they can represent biodiversity with a more useful macroecological variance to interplay with environmental variables. Moreover, because interaction predictions accounts only for <em>potential interactions</em>, they are easily translated into regional networks (that also represent potential interactions). Finally, some properties of networks are correlated to the phylogenetic diversity of communities [CHAP 1], which also represents a level of organization compatible with the geographical scale of environmental variables.</p>
            <h4 id="updating-probabilities-of-occurrences-with-network-probabilities">Updating probabilities of occurrences with network probabilities</h4>
            <p>At this point the reader may be wondering how probabilistic networks and species distribution models fit together. Some efforts in this sense have already been made, and in this session we will quickly review them while pointing to future directions. We will not consider, however, the majority of papers that aim at discussing the integration of biotic interactions on SDMs because their definitions of ecological interactions are not compatible with our view. Usually, these discussions consider the mere addition of the occurrence of a competitor or mutualist species as a predictor variable on the model to be sufficient to account for ecological interactions [<span class="citation" data-cites="Guisan2006MakBet">Guisan et al. (2006)</span>; <span class="citation" data-cites="Wisz2013RolBio">Wisz et al. (2013)</span>; CITATIONS]. Although this practice have demonstrated to improve the accuracy of distribution models [CITATIONS], the ecological interpretation is still not convincing. First, as we discussed earlier, the realization of an interaction is conditional to many factors, including abundance and other interactions, and it’s strength and signal can vary in space and time. Second, adding a competitor as an equivalent of an absence of the focal species just informs the model about environmental conditions that are supposedly not suitable for the establishment of a population, not that an ecological interaction is taking place there in such a manner that it will influence the survival of a pair of individuals.</p>
            <p>As reviewed by <span class="citation" data-cites="Cabral2017MecSim">Cabral, Valente, and Hartig (2017)</span>, many models in ecology have considered competition and facilitation in range shifts, but the assessment of trophic interactions remains insufficient. An example of good theoretical exercise was done in <span class="citation" data-cites="Godsoe2012HowSpe">Godsoe and Harmon (2012)</span>, where authors calculated the probability that a given site will be environmentally suitable in the presence of a competitor, based on their population dynamic models <span class="citation" data-cites="Godsoe2012HowSpe">(Godsoe and Harmon 2012)</span>. Although it is close to assuming a binary nature for interactions, it does account for dynamics and simulate results for different abundances, thus approximating to a probabilistic approach. There are only a few attempts on the literature that tries to build mechanistic species distribution models accounting for interspecies interaction (independent of its nature) as a probabilistic entity <span class="citation" data-cites="Cabral2017MecSim">(Cabral, Valente, and Hartig 2017)</span>. A first step in this direction was taken in <span class="citation" data-cites="Gravel2011TroThe">Gravel et al. (2011)</span>, where authors demonstrated how interactions can be included in the Theory of Island Biogeography <span class="citation" data-cites="MacArthur1967TheIsl">(MacArthur and Wilson 1967)</span>. This experiment provided a starting framework for the next generation of species distribution models to include migration dynamics and trophic levels.</p>
            <p>In <span class="citation" data-cites="Cazelles2016IntBio">Cazelles et al. (2016)</span>, the authors elaborate on <span class="citation" data-cites="Gravel2011TroThe">Gravel et al. (2011)</span> model and include environmental constraints in a metacommunities dynamic model in a biogeographical scale. Their model add some necessary complexity by evaluating frames of communities in different times, calculating the probability of change in community composition due to migration, interactions and environmental requirements. This is a very useful model to understand spatial dynamics of species richness, and could be used as a validation strategy for HSMs with only two time frames: the current known distribution and the potential distribution inferred by the HSM. Because the probability of occurrence of one species is derived from the probability of occurrence of all other species <span class="citation" data-cites="Cazelles2016IntBio">(Cazelles et al. 2016)</span>, we could infer the probability of occurrence of a focal species based on its potential distribution and local species pools. This strategy would require to predict interactions between species that did not previously co-occur.</p>
            <p>In this sense, an interesting approach was elaborated in <span class="citation" data-cites="Staniczenko2017LinMac">Staniczenko et al. (2017)</span>, where the authors address interactions with Bayesian Networks models that update the probability of occurrence resulted from a HSM. Bayesian Networks (BN) can be derived from empirical data or inferred from macroecological data, but they should only represent directional interactions. This was pointed as a limitation for the use of this workflow, as interactions are often retroactive [CITATION], but it showed good efficacy when the research subject allows this simplification <span class="citation" data-cites="Staniczenko2017LinMac">(Staniczenko et al. 2017)</span>. In these networks, the effects of direct and indirect interactions are spread among the species, and the nodes represent the probabilities of occurrence. This allows that multiple species are considered at the same time and to derive the network from the assemblage resulted of a HSM.</p>
            <h2 id="synthesis">Synthesis</h2>
            <p>We envision a combination of all techniques described here, which would allow a more reliable estimation of the area where a species would probably occur (illustrated as the area 2 in fig. 1 and in fig. 2). This combination would involve three steps, as delineated by <span class="citation" data-cites="Staniczenko2017LinMac">Staniczenko et al. (2017)</span>, with an initial HSM to assess the potential distribution of one or more focal species, and the use of machine learning algorithms to update the probability of occurrence from information on potential interactions. In this sense, there are two possible venues that could be followed: (i) calculate the probability of interaction from the local species pool, as suggest <span class="citation" data-cites="Cazelles2016IntBio">Cazelles et al. (2016)</span>, and use it to update the probabilities from the HSM, or (ii) calculate the probabilities of connections, then the probability of interactions where the chance of connection is higher than a threshold, and finally using this probability as a prior on a species distribution model. We propose a workflow that follows the later path, with four steps described below.</p>
            <figure>
            <img src="figures/conc_fig4_chapX.png" id="fig:synthesis" alt="Figure 2: In the proposed workflow, probabilities of occurrence would be updated by the probabilities of new interactions where the Habitat Suitability Model suggests potential occurrence. For the sample sites where a species is predicted to occur, but did not occur before, the probability of this species to belong to a network where it has at least one link would be calculated based on the species richness of those cells. For the cells where the probability of having at least one link is higher than 0 (or a given threshold, depending on the system), the probability of interaction with important clades would then be assessed, and finally this probability updates the probability of occurrence." /><figcaption aria-hidden="true">Figure 2: In the proposed workflow, probabilities of occurrence would be updated by the probabilities of new interactions where the Habitat Suitability Model suggests potential occurrence. For the sample sites where a species is predicted to occur, but did not occur before, the probability of this species to belong to a network where it has at least one link would be calculated based on the species richness of those cells. For the cells where the probability of having at least one link is higher than 0 (or a given threshold, depending on the system), the probability of interaction with important clades would then be assessed, and finally this probability updates the probability of occurrence.</figcaption>
            </figure>
            <h3 id="i-potential-distribution-of-focal-species">(i) Potential distribution of focal species</h3>
            <p>A traditional habitat suitability model should be used as a first step to assess the M+A space of the species (fig. 1). The choice of the best model will depend on a few criteria such as the nature and quality of your data, and the processes you want to investigate, as described in <span class="citation" data-cites="Guisan2017HabSui">Guisan, Thuiller, and Zimmermann (2017)</span> and <span class="citation" data-cites="Peterson2012EcoNic">Peterson et al. (2012)</span>. A HSM will indicate where there are suitable habitats for the species, which may include areas where the species was never sampled before. These areas, the “error” of the model, should be the object of assessment on the next steps since it is there where the unknown information about interactions is. For each sample cell where the species is predicted to occur, the value of known species richness can be used to calculate potential networks with the addition of the focal species.</p>
            <h3 id="ii-potential-networks-based-on-species-richness">(ii) Potential networks based on species richness</h3>
            <p>The minimum and maximum number of links that are possible depends on how many species share the same space <span class="citation" data-cites="MacDonald2020RevLin">(MacDonald, Banville, and Poisot 2020)</span>. Based on that, we can calculate the degree distributions that are possible based on Maximum Entropy <span class="citation" data-cites="francisbanville">(HALP <strong>francisbanville?</strong>)</span></p>
            <h3 id="iii-potential-interactions-where-the-probability-of-a-degree-ge-1-is-not-null">(iii) Potential interactions where the probability of a degree <span class="math inline">≥</span> 1 is not null</h3>
            <p>For the cells where the focal species has at least one possible network where its degree is <span class="math inline">≥</span> 1, we can proceed to investigate whether this link is ecologically relevant. This can be done by sampling a subset of species that occur and that could be important to that focal species (either a competitor, a prey, a predator or a facilitator). Some variables that can be important in this process are the phylogenetic or the functional diversity, since they can help us subset the species pool by evolutionary distance or ecological redundancy. Then the probability of interaction can be calculated for the focal species and the subset of species following the methods we discussed above, depending on the nature of the data available at a macroecological scale. This analysis should result on a spatial distribution of interaction probabilities for each cell where the species where predicted to occur but was not sampled before.</p>
            <h3 id="iv-probability-of-occurrence-given-the-probability-of-interactions">(iv) Probability of occurrence given the probability of interactions</h3>
            <p>For those cells where we now have the probability of occurrence and interaction, the probability of occurrence given the probability of interaction is a new function <em>h(x)</em>, which is a combination between a function of probability of occurrence <em>f(x)</em> and a function of probability of interaction <em>g(x)</em> eq. 1.<br />
            <span id="eq:bayup"><span class="math display"><em>h</em>(<em>x</em>) = <em>d</em>(<em>x</em>)⌊∫<em>f</em>(<em>x</em>)−(∫<em>f</em>(<em>x</em>)−<em>g</em>(<em>x</em>))−(∫<em>g</em>(<em>x</em>)−<em>f</em>(<em>x</em>))⌋   (1)</span></span></p>
            <p>TODO</p>
            <p>Updating the probabilities of occurrence based on the probabilities of interaction is a way to integrate biotic variables on species distribution models as a separate step. We believe that this pathway can simplify this intricate problem because it disentangles the complex calculation of ecological interactions from the abiotic suitability models. Both sides can be run in parallel, and in some cases the probability of interaction does not need to be calculated every time a model is run.</p>
            <ul>
            <li>What don’t we have yet?</li>
            <li>What are the roads never taken?</li>
            <li>Where should we invest in order to achieve the best SDMs we can?</li>
            <li>What are the most promising areas for development?</li>
            </ul>
            <h2 class="unnumbered" id="references">References</h2>
            <div id="refs" class="references csl-bib-body hanging-indent" role="doc-bibliography">
            <div id="ref-Afkhami2014MutEff" class="csl-entry" role="doc-biblioentry">
            Afkhami, Michelle E., Patrick J. McIntyre, and Sharon Y. Strauss. 2014. <span>“Mutualist-Mediated Effects on Species’ Range Limits Across Large Geographic Scales.”</span> <em>Ecology Letters</em> 17 (10): 1265–73. <a href="https://doi.org/10.1111/ele.12332">https://doi.org/10.1111/ele.12332</a>.
            </div>
            <div id="ref-Bar-Massada2017NonCoo" class="csl-entry" role="doc-biblioentry">
            Bar-Massada, Avi, and Jonathan Belmaker. 2017. <span>“Non-Stationarity in the Co-Occurrence Patterns of Species Across Environmental Gradients.”</span> <em>Journal of Ecology</em> 105 (2): 391–99. <a href="https://doi.org/10.1111/1365-2745.12713">https://doi.org/10.1111/1365-2745.12713</a>.
            </div>
            <div id="ref-Blanchet2020CooNot" class="csl-entry" role="doc-biblioentry">
            Blanchet, F. Guillaume, Kevin Cazelles, and Dominique Gravel. 2020. <span>“Co-Occurrence Is Not Evidence of Ecological Interactions.”</span> <em>Ecology Letters</em> 23 (7): 1050–63. <a href="https://doi.org/10.1111/ele.13525">https://doi.org/10.1111/ele.13525</a>.
            </div>
            <div id="ref-Breiman2001RanFor" class="csl-entry" role="doc-biblioentry">
            Breiman, Leo. 2001. <span>“Random Forests.”</span> <em>Machine Learning</em> 45 (1): 5–32. <a href="https://doi.org/10.1023/A:1010933404324">https://doi.org/10.1023/A:1010933404324</a>.
            </div>
            <div id="ref-Bullock2000GeoSep" class="csl-entry" role="doc-biblioentry">
            Bullock, James M., Rebecca J. Edwards, Peter D. Carey, and Rob J. Rose. 2000. <span>“Geographical Separation of Two Ulex Species at Three Spatial Scales: Does Competition Limit Species’ Ranges?”</span> <em>Ecography</em> 23 (2): 257–71. <a href="https://doi.org/10.1111/j.1600-0587.2000.tb00281.x">https://doi.org/10.1111/j.1600-0587.2000.tb00281.x</a>.
            </div>
            <div id="ref-Cabral2017MecSim" class="csl-entry" role="doc-biblioentry">
            Cabral, Juliano Sarmento, Luis Valente, and Florian Hartig. 2017. <span>“Mechanistic Simulation Models in Macroecology and Biogeography: State-of-Art and Prospects.”</span> <em>Ecography</em> 40 (2): 267–80. <a href="https://doi.org/10.1111/ecog.02480">https://doi.org/10.1111/ecog.02480</a>.
            </div>
            <div id="ref-Callaghan2019ImpBig" class="csl-entry" role="doc-biblioentry">
            Callaghan, Corey T., Jodi J. L. Rowley, William K. Cornwell, Alistair G. B. Poore, and Richard E. Major. 2019. <span>“Improving Big Citizen Science Data: Moving Beyond Haphazard Sampling.”</span> <em>PLOS Biology</em> 17 (6): e3000357. <a href="https://doi.org/10.1371/journal.pbio.3000357">https://doi.org/10.1371/journal.pbio.3000357</a>.
            </div>
            <div id="ref-Canard2012EmeStr" class="csl-entry" role="doc-biblioentry">
            Canard, Elsa, Nicolas Mouquet, Lucile Marescot, Kevin J. Gaston, Dominique Gravel, and David Mouillot. 2012. <span>“Emergence of Structural Patterns in Neutral Trophic Networks.”</span> <em>PLOS ONE</em> 7 (8): e38295. <a href="https://doi.org/10.1371/journal.pone.0038295">https://doi.org/10.1371/journal.pone.0038295</a>.
            </div>
            <div id="ref-Carnicer2009TemDyn" class="csl-entry" role="doc-biblioentry">
            Carnicer, Jofre, Pedro Jordano, and Carlos J Melián. 2009. <span>“The Temporal Dynamics of Resource Use by Frugivorous Birds: A Network Approach.”</span> <em>Ecology</em> 90 (7): 1958–70.
            </div>
            <div id="ref-Cazelles2016IntBio" class="csl-entry" role="doc-biblioentry">
            Cazelles, Kévin, Nicolas Mouquet, David Mouillot, and Dominique Gravel. 2016. <span>“On the Integration of Biotic Interaction and Environmental Constraints at the Biogeographical Scale.”</span> <em>Ecography</em> 39 (10): 921–31. <a href="https://doi.org/10.1111/ecog.01714">https://doi.org/10.1111/ecog.01714</a>.
            </div>
            <div id="ref-Chesson2008IntPre" class="csl-entry" role="doc-biblioentry">
            Chesson, Peter, and Jessica J. Kuang. 2008. <span>“The Interaction Between Predation and Competition.”</span> <em>Nature</em> 456 (7219): 235–38. <a href="https://doi.org/10.1038/nature07248">https://doi.org/10.1038/nature07248</a>.
            </div>
            <div id="ref-Coelho2017NeuBio" class="csl-entry" role="doc-biblioentry">
            Coelho, Marco Túlio Pacheco, João Fabrício Mota Rodrigues, and Thiago F Rangel. 2017. <span>“Neutral Biogeography of Phylogenetically Structured Interaction Networks.”</span> <em>Ecography</em> 40 (12): 1467–74.
            </div>
            <div id="ref-Dallas2017PreCry" class="csl-entry" role="doc-biblioentry">
            Dallas, Tad, Andrew W. Park, and John M. Drake. 2017. <span>“Predicting Cryptic Links in Host-Parasite Networks.”</span> <em>PLOS Computational Biology</em> 13 (5): e1005557. <a href="https://doi.org/10.1371/journal.pcbi.1005557">https://doi.org/10.1371/journal.pcbi.1005557</a>.
            </div>
            <div id="ref-Dallas2018ComTur" class="csl-entry" role="doc-biblioentry">
            Dallas, Tad, and Timothée Poisot. 2018. <span>“Compositional Turnover in Host and Parasite Communities Does Not Change Network Structure.”</span> <em>Ecography</em> 41 (9): 1534–42. <a href="https://doi.org/10.1111/ecog.03514">https://doi.org/10.1111/ecog.03514</a>.
            </div>
            <div id="ref-Dalsgaard2013HisCli" class="csl-entry" role="doc-biblioentry">
            Dalsgaard, Bo, Kristian Trøjelsgaard, Ana M Martín González, David Nogués-Bravo, Jeff Ollerton, Theodora Petanidou, Brody Sandel, et al. 2013. <span>“Historical Climate-Change Influences Modularity and Nestedness of Pollination Networks.”</span> <em>Ecography</em> 36 (12): 1331–40.
            </div>
            <div id="ref-Davies2011PhyDiv" class="csl-entry" role="doc-biblioentry">
            Davies, T Jonathan, and Lauren B Buckley. 2011. <span>“Phylogenetic Diversity as a Window into the Evolutionary and Biogeographic Histories of Present-Day Richness Gradients for Mammals.”</span> <em>Philos. Trans. R. Soc. Lond. B Biol. Sci.</em> 366 (1576): 2414–25.
            </div>
            <div id="ref-Dattilo2020SpeDri" class="csl-entry" role="doc-biblioentry">
            Dáttilo, Wesley, Nathalia Barrozo-Chávez, Andrés Lira-Noriega, Roger Guevara, Fabricio Villalobos, Diego Santiago-Alarcon, Frederico Siqueira Neves, Thiago Izzo, and Sérvio Pontes Ribeiro. 2020. <span>“Species-Level Drivers of Mammalian Ectoparasite Faunas.”</span> <em>Journal of Animal Ecology</em> 89 (8): 1754–65. <a href="https://doi.org/10.1111/1365-2656.13216">https://doi.org/10.1111/1365-2656.13216</a>.
            </div>
            <div id="ref-Desjardins-Proulx2017EcoInt" class="csl-entry" role="doc-biblioentry">
            Desjardins-Proulx, Philippe, Idaline Laigle, Timothée Poisot, and Dominique Gravel. 2017. <span>“Ecological Interactions and the Netflix Problem.”</span> <em>PeerJ</em> 5 (August): e3644. <a href="https://doi.org/10.7717/peerj.3644">https://doi.org/10.7717/peerj.3644</a>.
            </div>
            <div id="ref-Dunne2002NetStr" class="csl-entry" role="doc-biblioentry">
            Dunne, Jennifer A., Richard J. Williams, and Neo D. Martinez. 2002. <span>“Network Structure and Biodiversity Loss in Food Webs: Robustness Increases with Connectance.”</span> <em>Ecology Letters</em> 5 (4): 558–67. <a href="https://doi.org/10.1046/j.1461-0248.2002.00354.x">https://doi.org/10.1046/j.1461-0248.2002.00354.x</a>.
            </div>
            <div id="ref-Elmasri2020HieBay" class="csl-entry" role="doc-biblioentry">
            Elmasri, Mohamad, Maxwell J. Farrell, T. Jonathan Davies, and David A. Stephens. 2020. <span>“A Hierarchical Bayesian Model for Predicting Ecological Interactions Using Scaled Evolutionary Relationships.”</span> <em>Annals of Applied Statistics</em> 14 (1): 221–40. <a href="https://doi.org/10.1214/19-AOAS1296">https://doi.org/10.1214/19-AOAS1296</a>.
            </div>
            <div id="ref-Ferrier2006SpaMod" class="csl-entry" role="doc-biblioentry">
            Ferrier, Simon, and Antoine Guisan. 2006. <span>“Spatial Modelling of Biodiversity at the Community Level.”</span> <em>Journal of Applied Ecology</em> 43 (3): 393–404. <a href="https://doi.org/10.1111/j.1365-2664.2006.01149.x">https://doi.org/10.1111/j.1365-2664.2006.01149.x</a>.
            </div>
            <div id="ref-Galiana2018SpaSca" class="csl-entry" role="doc-biblioentry">
            Galiana, Nuria, Miguel Lurgi, Bernat Claramunt-López, Marie-Josée Fortin, Shawn Leroux, Kevin Cazelles, Dominique Gravel, and José M. Montoya. 2018. <span>“The Spatial Scaling of Species Interaction Networks.”</span> <em>Nature Ecology &amp; Evolution</em> 2 (5): 782–90. <a href="https://doi.org/10.1038/s41559-018-0517-3">https://doi.org/10.1038/s41559-018-0517-3</a>.
            </div>
            <div id="ref-Gbi" class="csl-entry" role="doc-biblioentry">
            <span>“GBIF.”</span> n.d. https://www.gbif.org/.
            </div>
            <div id="ref-Godsoe2012HowSpe" class="csl-entry" role="doc-biblioentry">
            Godsoe, William, and Luke J. Harmon. 2012. <span>“How Do Species Interactions Affect Species Distribution Models?”</span> <em>Ecography</em> 35 (9): 811–20. <a href="https://doi.org/10.1111/j.1600-0587.2011.07103.x">https://doi.org/10.1111/j.1600-0587.2011.07103.x</a>.
            </div>
            <div id="ref-Godsoe2017IntBio" class="csl-entry" role="doc-biblioentry">
            Godsoe, William, Jill Jankowski, Robert D. Holt, and Dominique Gravel. 2017. <span>“Integrating Biogeography with Contemporary Niche Theory.”</span> <em>Trends in Ecology and Evolution</em> 32 (7): 488–99. <a href="https://doi.org/10.1016/j.tree.2017.03.008">https://doi.org/10.1016/j.tree.2017.03.008</a>.
            </div>
            <div id="ref-Gomez2010EcoInt" class="csl-entry" role="doc-biblioentry">
            Gómez, José M., Miguel Verdú, and Francisco Perfectti. 2010. <span>“Ecological Interactions Are Evolutionarily Conserved Across the Entire Tree of Life.”</span> <em>Nature</em> 465 (7300): 918–21. <a href="https://doi.org/10.1038/nature09113">https://doi.org/10.1038/nature09113</a>.
            </div>
            <div id="ref-Gravel2011TroThe" class="csl-entry" role="doc-biblioentry">
            Gravel, Dominique, François Massol, Elsa Canard, David Mouillot, and Nicolas Mouquet. 2011. <span>“Trophic Theory of Island Biogeography.”</span> <em>Ecology Letters</em> 14 (10): 1010–16. <a href="https://doi.org/10.1111/j.1461-0248.2011.01667.x">https://doi.org/10.1111/j.1461-0248.2011.01667.x</a>.
            </div>
            <div id="ref-Guimaraes2020StrEco" class="csl-entry" role="doc-biblioentry">
            Guimarães, Paulo R. 2020. <span>“The Structure of Ecological Networks Across Levels of Organization.”</span> <em>Annual Review of Ecology, Evolution, and Systematics</em> 51 (1): 433–60. <a href="https://doi.org/10.1146/annurev-ecolsys-012220-120819">https://doi.org/10.1146/annurev-ecolsys-012220-120819</a>.
            </div>
            <div id="ref-Guisan2006MakBet" class="csl-entry" role="doc-biblioentry">
            Guisan, Antoine, Anthony Lehmann, Simon Ferrier, Mike Austin, Jacob Mc. C. Overton, Richard Aspinall, and Trevor Hastie. 2006. <span>“Making Better Biogeographical Predictions of Species’ Distributions.”</span> <em>Journal of Applied Ecology</em> 43 (3): 386–92. <a href="https://doi.org/10.1111/j.1365-2664.2006.01164.x">https://doi.org/10.1111/j.1365-2664.2006.01164.x</a>.
            </div>
            <div id="ref-Guisan2017HabSui" class="csl-entry" role="doc-biblioentry">
            Guisan, Antoine, Wilfried Thuiller, and Niklaus E. Zimmermann. 2017. <em>Habitat Suitability and Distribution Models: With Applications in R</em>. Ecology, Biodiversity and Conservation. Cambridge: Cambridge University Press. <a href="https://doi.org/10.1017/9781139028271">https://doi.org/10.1017/9781139028271</a>.
            </div>
            <div id="ref-Guisan2000PreHab" class="csl-entry" role="doc-biblioentry">
            Guisan, Antoine, and Niklaus E. Zimmermann. 2000. <span>“Predictive Habitat Distribution Models in Ecology.”</span> <em>Ecological Modelling</em> 135 (2): 147–86. <a href="https://doi.org/10.1016/S0304-3800(00)00354-9">https://doi.org/10.1016/S0304-3800(00)00354-9</a>.
            </div>
            <div id="ref-Harmon2010EarBur" class="csl-entry" role="doc-biblioentry">
            Harmon, Luke J., Jonathan B. Losos, T. Jonathan Davies, Rosemary G. Gillespie, John L. Gittleman, W. Bryan Jennings, Kenneth H. Kozak, et al. 2010. <span>“Early Bursts of Body Size and Shape Evolution Are Rare in Comparative Data.”</span> <em>Evolution</em> 64 (8): 2385–96. <a href="https://doi.org/10.1111/j.1558-5646.2010.01025.x">https://doi.org/10.1111/j.1558-5646.2010.01025.x</a>.
            </div>
            <div id="ref-Hellmann2012InfSpe" class="csl-entry" role="doc-biblioentry">
            Hellmann, Jessica J., Kirsten M. Prior, and Shannon L. Pelini. 2012. <span>“The Influence of Species Interactions on Geographic Range Change Under Climate Change.”</span> <em>Annals of the New York Academy of Sciences</em> 1249 (February): 18–28. <a href="https://doi.org/10.1111/j.1749-6632.2011.06410.x">https://doi.org/10.1111/j.1749-6632.2011.06410.x</a>.
            </div>
            <div id="ref-Hortal2015SevSho" class="csl-entry" role="doc-biblioentry">
            Hortal, Joaquín, Francesco de Bello, José Alexandre F. Diniz-Filho, Thomas M. Lewinsohn, Jorge M. Lobo, and Richard J. Ladle. 2015. <span>“Seven Shortfalls That Beset Large-Scale Knowledge of Biodiversity.”</span> <em>Annual Review of Ecology, Evolution, and Systematics</em> 46 (1): 523–49. <a href="https://doi.org/10.1146/annurev-ecolsys-112414-054400">https://doi.org/10.1146/annurev-ecolsys-112414-054400</a>.
            </div>
            <div id="ref-Ingram2012WheSho" class="csl-entry" role="doc-biblioentry">
            Ingram, T., L. J. Harmon, and J. B. Shurin. 2012. <span>“When Should We Expect Early Bursts of Trait Evolution in Comparative Data? Predictions from an Evolutionary Food Web Model.”</span> <em>Journal of Evolutionary Biology</em> 25 (9): 1902–10. <a href="https://doi.org/10.1111/j.1420-9101.2012.02566.x">https://doi.org/10.1111/j.1420-9101.2012.02566.x</a>.
            </div>
            <div id="ref-Jordano2016SamNet" class="csl-entry" role="doc-biblioentry">
            Jordano, Pedro. 2016. <span>“Sampling Networks of Ecological Interactions.”</span> <em>Functional Ecology</em> 30 (12): 1883–93. <a href="https://doi.org/10.1111/1365-2435.12763">https://doi.org/10.1111/1365-2435.12763</a>.
            </div>
            <div id="ref-JorgeSoberon2007GriElt" class="csl-entry" role="doc-biblioentry">
            Jorge Soberón. 2007. <span>“Grinnellian and Eltonian Niches and Geographic Distributions of Species.”</span> <em>Ecology Letters</em> 10 (12): 1115–23. <a href="https://doi.org/10.1111/j.1461-0248.2007.01107.x">https://doi.org/10.1111/j.1461-0248.2007.01107.x</a>.
            </div>
            <div id="ref-Konig2019BioDat" class="csl-entry" role="doc-biblioentry">
            König, Christian, Patrick Weigelt, Julian Schrader, Amanda Taylor, Jens Kattge, and Holger Kreft. 2019. <span>“Biodiversity Data Integrationthe Significance of Data Resolution and Domain.”</span> <em>PLOS Biology</em> 17 (3): e3000183. <a href="https://doi.org/10.1371/journal.pbio.3000183">https://doi.org/10.1371/journal.pbio.3000183</a>.
            </div>
            <div id="ref-MacArthur1967TheIsl" class="csl-entry" role="doc-biblioentry">
            MacArthur, Robert H., and Edward O. Wilson. 1967. <em>The Theory of Island Biogeography</em>. Princeton University Press.
            </div>
            <div id="ref-MacDonald2020RevLin" class="csl-entry" role="doc-biblioentry">
            MacDonald, Arthur Andrew Meahan, Francis Banville, and Timothée Poisot. 2020. <span>“Revisiting the Links-Species Scaling Relationship in Food Webs.”</span> <em>Patterns</em> 1 (0). <a href="https://doi.org/10.1016/j.patter.2020.100079">https://doi.org/10.1016/j.patter.2020.100079</a>.
            </div>
            <div id="ref-MartinGonzalez2015MacPhy" class="csl-entry" role="doc-biblioentry">
            Martín González, Ana M., Bo Dalsgaard, David Nogués-Bravo, Catherine H. Graham, Matthias Schleuning, Pietro K. Maruyama, Stefan Abrahamczyk, et al. 2015. <span>“The Macroecology of Phylogenetically Structured Hummingbird-Plant Networks.”</span> <em>Global Ecology and Biogeography</em> 24 (11): 1212–24. <a href="https://doi.org/10.1111/geb.12355">https://doi.org/10.1111/geb.12355</a>.
            </div>
            <div id="ref-Memmott2004TolPol" class="csl-entry" role="doc-biblioentry">
            Memmott, J., N. M. Waser, and M. V. Price. 2004. <span>“Tolerance of Pollination Networks to Species Extinctions.”</span> <em>Proceedings of the Royal Society B: Biological Sciences</em> 271 (1557): 2605–11. <a href="https://doi.org/10.1098/rspb.2004.2909">https://doi.org/10.1098/rspb.2004.2909</a>.
            </div>
            <div id="ref-Muola2010AssPla" class="csl-entry" role="doc-biblioentry">
            Muola, Anne, Pia Mutikainen, Marianna Lilley, Liisa Laukkanen, Juha Pekka Salminen, and Roosa Leimu. 2010. <span>“Associations of Plant Fitness, Leaf Chemistry, and Damage Suggest Selection Mosaic in Plant-Herbivore Interactions.”</span> <em>Ecology</em>. <a href="https://doi.org/10.1890/09-0589.1">https://doi.org/10.1890/09-0589.1</a>.
            </div>
            <div id="ref-Olden2008MacLea" class="csl-entry" role="doc-biblioentry">
            Olden, Julian, Joshua Lawler, and N. Poff. 2008. <span>“Machine Learning Methods Without Tears: A Primer for Ecologists.”</span> <em>The Quarterly Review of Biology</em> 83 (July): 171–93. <a href="https://doi.org/10.1086/587826">https://doi.org/10.1086/587826</a>.
            </div>
            <div id="ref-Peralta2016MerEvo" class="csl-entry" role="doc-biblioentry">
            Peralta, Guadalupe. 2016. <span>“Merging Evolutionary History into Species Interaction Networks.”</span> <em>Functional Ecology</em> 30 (12): 1917–25. <a href="https://doi.org/10.1111/1365-2435.12669">https://doi.org/10.1111/1365-2435.12669</a>.
            </div>
            <div id="ref-Peterson2012EcoNic" class="csl-entry" role="doc-biblioentry">
            Peterson, A. Townsend, Jorge Soberón, Richard G. Pearson, Robert P. Anderson, Enrique Martínez-Meyer, Miguel Nakamura, and Miguel Bastos Araújo. 2012. <em>Ecological Niches and Geographic Distributions</em>. <em>Choice Reviews Online</em>. Vol. 49. <a href="https://doi.org/10.5860/CHOICE.49-6266">https://doi.org/10.5860/CHOICE.49-6266</a>.
            </div>
            <div id="ref-Phillips2006MaxEnt" class="csl-entry" role="doc-biblioentry">
            Phillips, Steven J., Robert P. Anderson, and Robert E. Schapire. 2006. <span>“Maximum Entropy Modeling of Species Geographic Distributions.”</span> <em>Ecological Modelling</em> 190 (3): 231–59. <a href="https://doi.org/10.1016/j.ecolmodel.2005.03.026">https://doi.org/10.1016/j.ecolmodel.2005.03.026</a>.
            </div>
            <div id="ref-Pichler2019MacLea" class="csl-entry" role="doc-biblioentry">
            Pichler, Maximilian, Virginie Boreux, Alexandra-Maria Klein, Matthias Schleuning, and Florian Hartig. <span>“Machine Learning Algorithms to Infer Trait-Matching and Predict Species Interactions in Ecological Networks.”</span> <em>Methods in Ecology and Evolution</em> 11 (2): 281–93. <a href="https://doi.org/10.1111/2041-210X.13329">https://doi.org/10.1111/2041-210X.13329</a>.
            </div>
            <div id="ref-Pocock2015BioRec" class="csl-entry" role="doc-biblioentry">
            Pocock, Michael J. O., Helen E. Roy, Chris D. Preston, and David B. Roy. 2015. <span>“The Biological Records Centre: A Pioneer of Citizen Science.”</span> <em>Biological Journal of the Linnean Society</em> 115 (3): 475–93. <a href="https://doi.org/10.1111/bij.12548">https://doi.org/10.1111/bij.12548</a>.
            </div>
            <div id="ref-Poisot2016ManMak" class="csl-entry" role="doc-biblioentry">
            Poisot, Timothée, Benjamin Baiser, Jennifer A Dunne, Sonia Kéfi, François Massol, Nicolas Mouquet, Tamara N Romanuk, Daniel B Stouffer, Spencer A Wood, and Dominique Gravel. 2016. <span>“Mangal - Making Ecological Network Analysis Simple.”</span> <em>Ecography</em> 39 (4): 384–90.
            </div>
            <div id="ref-Poisot2020EnvBia" class="csl-entry" role="doc-biblioentry">
            Poisot, Timothée, Gabriel Bergeron, Kevin Cazelles, Tad Dallas, Dominique Gravel, Andrew Macdonald, Benjamin Mercier, Clément Violet, and Steve Vissault. 2020. <span>“Environmental Biases in the Study of Ecological Networks at the Planetary Scale.”</span> <em>bioRxiv</em>, January. <a href="https://doi.org/10.1101/2020.01.27.921429">https://doi.org/10.1101/2020.01.27.921429</a>.
            </div>
            <div id="ref-Poisot2016StrPro" class="csl-entry" role="doc-biblioentry">
            Poisot, Timothée, Alyssa R Cirtwill, Kévin Cazelles, Dominique Gravel, Marie Josée Fortin, and Daniel B Stouffer. 2016. <span>“The Structure of Probabilistic Networks.”</span> <em>Methods in Ecology and Evolution</em> 7 (3): 303–12. <a href="https://doi.org/10.1111/2041-210X.12468">https://doi.org/10.1111/2041-210X.12468</a>.
            </div>
            <div id="ref-Poisot2017HosPar" class="csl-entry" role="doc-biblioentry">
            Poisot, Timothée, Cynthia Guéveneux-Julien, Marie Josée Fortin, Dominique Gravel, and Pierre Legendre. 2017. <span>“Hosts, Parasites and Their Interactions Respond to Different Climatic Variables.”</span> <em>Global Ecology and Biogeography</em> 26 (8): 942–51. <a href="https://doi.org/10.1111/geb.12602">https://doi.org/10.1111/geb.12602</a>.
            </div>
            <div id="ref-Poisot2016HowEco" class="csl-entry" role="doc-biblioentry">
            Poisot, Timothée, and Daniel Stouffer. 2016. <span>“How Ecological Networks Evolve.”</span> <em>bioRxiv</em>, no. Jablonski 2008: 071993. <a href="https://doi.org/10.1101/071993">https://doi.org/10.1101/071993</a>.
            </div>
            <div id="ref-Poisot2014SpeWhy" class="csl-entry" role="doc-biblioentry">
            Poisot, Timothée, Daniel B. Stouffer, and Dominique Gravel. 2014. <span>“Beyond Species: Why Ecological Interaction Networks Vary Through Space and Time.”</span> <em>Oikos</em> 124 (3): 243–51. <a href="https://doi.org/10.1111/oik.01719">https://doi.org/10.1111/oik.01719</a>.
            </div>
            <div id="ref-Pollock2014UndCoo" class="csl-entry" role="doc-biblioentry">
            Pollock, Laura J., Reid Tingley, William K. Morris, Nick Golding, Robert B. O’Hara, Kirsten M. Parris, Peter A. Vesk, and Michael A. McCarthy. 2014. <span>“Understanding Co-Occurrence by Modelling Species Simultaneously with a Joint Species Distribution Model (JSDM).”</span> <em>Methods in Ecology and Evolution</em> 5 (5): 397–406. <a href="https://doi.org/10.1111/2041-210X.12180">https://doi.org/10.1111/2041-210X.12180</a>.
            </div>
            <div id="ref-Rall2012UniTem" class="csl-entry" role="doc-biblioentry">
            Rall, B. C., U. Brose, M. Hartvig, G. Kalinkat, F. Schwarzmuller, O. Vucic-Pestic, and O. L. Petchey. 2012. <span>“Universal Temperature and Body-Mass Scaling of Feeding Rates.”</span> <em>Philosophical Transactions of the Royal Society B: Biological Sciences</em> 367 (1605): 2923–34. <a href="https://doi.org/10.1098/rstb.2012.0242">https://doi.org/10.1098/rstb.2012.0242</a>.
            </div>
            <div id="ref-Ramos-Jiliberto2012TopPla" class="csl-entry" role="doc-biblioentry">
            Ramos-Jiliberto, Rodrigo, Fernanda S. Valdovinos, Pablo Moisset de Espanés, and José D. Flores. 2012. <span>“Topological Plasticity Increases Robustness of Mutualistic Networks.”</span> <em>Journal of Animal Ecology</em> 81 (4): 896–904. <a href="https://doi.org/10.1111/j.1365-2656.2012.01960.x">https://doi.org/10.1111/j.1365-2656.2012.01960.x</a>.
            </div>
            <div id="ref-Rangel2012LabEco" class="csl-entry" role="doc-biblioentry">
            Rangel, Thiago Fernando, and Rafael Dias Loyola. 2012. <span>“Labeling Ecological Niche Models.”</span> <em>Natureza &amp; Conservação</em> 10 (2): 119–26. <a href="https://doi.org/10.4322/natcon.2012.030">https://doi.org/10.4322/natcon.2012.030</a>.
            </div>
            <div id="ref-Riva2016ExpEvo" class="csl-entry" role="doc-biblioentry">
            Riva, Giulio V. Dalla, and Daniel B. Stouffer. 2016. <span>“Exploring the Evolutionary Signature of Food Webs’ Backbones Using Functional Traits.”</span> <em>Oikos</em> 125 (4): 446–56. <a href="https://doi.org/10.1111/oik.02305">https://doi.org/10.1111/oik.02305</a>.
            </div>
            <div id="ref-Roy2016FocPla" class="csl-entry" role="doc-biblioentry">
            Roy, Helen E., Elizabeth Baxter, Aoine Saunders, and Michael J. O. Pocock. 2016. <span>“Focal Plant Observations as a Standardised Method for Pollinator Monitoring: Opportunities and Limitations for Mass Participation Citizen Science.”</span> <em>PLOS ONE</em> 11 (3): e0150794. <a href="https://doi.org/10.1371/journal.pone.0150794">https://doi.org/10.1371/journal.pone.0150794</a>.
            </div>
            <div id="ref-Ryan2018RolCit" class="csl-entry" role="doc-biblioentry">
            Ryan, S. F., N. L. Adamson, A. Aktipis, L. K. Andersen, R. Austin, L. Barnes, M. R. Beasley, et al. 2018. <span>“The Role of Citizen Science in Addressing Grand Challenges in Food and Agriculture Research.”</span> <em>Proceedings of the Royal Society B: Biological Sciences</em> 285 (1891). <a href="https://doi.org/10.1098/rspb.2018.1977">https://doi.org/10.1098/rspb.2018.1977</a>.
            </div>
            <div id="ref-Sanders2012IndCom" class="csl-entry" role="doc-biblioentry">
            Sanders, Dirk, and F. J.Frank Van Veen. 2012. <span>“Indirect Commensalism Promotes Persistence of Secondary Consumer Species.”</span> <em>Biology Letters</em> 8 (6): 960–63. <a href="https://doi.org/10.1098/rsbl.2012.0572">https://doi.org/10.1098/rsbl.2012.0572</a>.
            </div>
            <div id="ref-Sebastian-Gonzalez2015MacTre" class="csl-entry" role="doc-biblioentry">
            Sebastián-González, Esther, Bo Dalsgaard, Brody Sandel, and Paulo R Guimarães. 2015. <span>“Macroecological Trends in Nestedness and Modularity of Seed-Dispersal Networks: Human Impact Matters.”</span> <em>Global Ecology and Biogeography</em> 24 (3): 293–303.
            </div>
            <div id="ref-Siren2020IntRan" class="csl-entry" role="doc-biblioentry">
            Sirén, Alexej P. K., and Toni Lyn Morelli. 2020. <span>“Interactive Range-Limit Theory (iRLT): An Extension for Predicting Range Shifts.”</span> <em>Journal of Animal Ecology</em> 89 (4): 940–54. <a href="https://doi.org/10.1111/1365-2656.13150">https://doi.org/10.1111/1365-2656.13150</a>.
            </div>
            <div id="ref-Soberon2009NicDis" class="csl-entry" role="doc-biblioentry">
            Soberón, Jorge, and Miguel Nakamura. 2009. <span>“Niches and Distributional Areas: Concepts, Methods, and Assumptions.”</span> <em>Proceedings of the National Academy of Sciences</em> 106 (Supplement 2): 19644–50. <a href="https://doi.org/10.1073/pnas.0901637106">https://doi.org/10.1073/pnas.0901637106</a>.
            </div>
            <div id="ref-Staniczenko2017LinMac" class="csl-entry" role="doc-biblioentry">
            Staniczenko, Phillip P. A., Prabu Sivasubramaniam, K. Blake Suttle, and Richard G. Pearson. 2017. <span>“Linking Macroecology and Community Ecology: Refining Predictions of Species Distributions Using Biotic Interaction Networks.”</span> <em>Ecology Letters</em> 20 (6): 693–707. <a href="https://doi.org/10.1111/ele.12770">https://doi.org/10.1111/ele.12770</a>.
            </div>
            <div id="ref-Svenning2014InfInt" class="csl-entry" role="doc-biblioentry">
            Svenning, Jens Christian, Dominique Gravel, Robert D. Holt, Frank M. Schurr, Wilfried Thuiller, Tamara Münkemüller, Katja H. Schiffers, et al. 2014. <span>“The Influence of Interspecific Interactions on Species Range Expansion Rates.”</span> <em>Ecography</em> 37 (12): 1198–1209. <a href="https://doi.org/10.1111/j.1600-0587.2013.00574.x">https://doi.org/10.1111/j.1600-0587.2013.00574.x</a>.
            </div>
            <div id="ref-Trojelsgaard2013MacPol" class="csl-entry" role="doc-biblioentry">
            Trøjelsgaard, Kristian, and Jens M Olesen. 2013. <span>“Macroecology of Pollination Networks.”</span> <em>Global Ecology and Biogeography</em> 22 (2): 149–62.
            </div>
            <div id="ref-Tylianakis2017EcoNet" class="csl-entry" role="doc-biblioentry">
            Tylianakis, Jason M., and Rebecca J. Morris. 2017. <span>“Ecological Networks Across Environmental Gradients.”</span> <em>Annual Review of Ecology, Evolution, and Systematics</em> 48 (1): annurev-ecolsys-110316-022821. <a href="https://doi.org/10.1146/annurev-ecolsys-110316-022821">https://doi.org/10.1146/annurev-ecolsys-110316-022821</a>.
            </div>
            <div id="ref-Wisz2013RolBio" class="csl-entry" role="doc-biblioentry">
            Wisz, Mary Susanne, Julien Pottier, W Daniel Kissling, Loïc Pellissier, Jonathan Lenoir, Christian F Damgaard, Carsten F Dormann, et al. 2013. <span>“The Role of Biotic Interactions in Shaping Distributions and Realised Assemblages of Species: Implications for Species Distribution Modelling.”</span> <em>Biological Reviews of the Cambridge Philosophical Society</em> 88 (1): 15–30. <a href="https://doi.org/10.1111/j.1469-185X.2012.00235.x">https://doi.org/10.1111/j.1469-185X.2012.00235.x</a>.
            </div>
            <div id="ref-Wolin1984ModFac" class="csl-entry" role="doc-biblioentry">
            Wolin, Carole L., and Lawrence R. Lawlor. 1984. <span>“Models of Facultative Mutualism: Density Effects.”</span> <em>The American Naturalist</em> 124 (6): 843–62. <a href="https://doi.org/10.1086/284320">https://doi.org/10.1086/284320</a>.
            </div>
            <div id="ref-XiaoFu2019LinPre" class="csl-entry" role="doc-biblioentry">
            Xiao Fu, Eugene Seo, Justin Clarke, and Rebecca A. Hutchinson. 2019. <span>“Link Prediction Under Imperfect Detection: Collaborative Filtering for Ecological Networks.”</span> <em>IEEE Transactions on Knowledge &amp; Data Engineering</em>, no. 01 (December). <a href="https://doi.org/10.1109/TKDE.2019.2962031">https://doi.org/10.1109/TKDE.2019.2962031</a>.
            </div>
            <div id="ref-Zurell2020TesSpe" class="csl-entry" role="doc-biblioentry">
            Zurell, Damaris, Niklaus E. Zimmermann, Helge Gross, Andri Baltensweiler, Thomas Sattler, and Rafael O. Wüest. 2020. <span>“Testing Species Assemblage Predictions from Stacked and Joint Species Distribution Models.”</span> <em>Journal of Biogeography</em> 47 (1): 101–13. <a href="https://doi.org/10.1111/jbi.13608">https://doi.org/10.1111/jbi.13608</a>.
            </div>
            </div>
        </article>
    </main>

    <footer>
        <div class="container">
            <h2>Affiliations</h2>
                        <div><sup>1</sup> Universidade Federal de Goiás</div>
                        <div><sup>2</sup> Université de Montréal</div>
                        <div><sup>3</sup> Université de Sherbrooke</div>
                        <div><sup>4</sup> Québec Centre for Biodiversity Sciences</div>
                    </div>
    </footer>


</body>
<script src="//cdnjs.cloudflare.com/ajax/libs/less.js/3.9.0/less.min.js"></script>

</html>
